validate_email/regex_check.py

```diff
@@ -27,41 +25,26 @@ def _validate_ipv6_address(value: str) -> bool:
 
 
 def _validate_ipv46_address(value: str) -> bool:
-    if _validate_ipv4_address(value):
-        return True
-    return _validate_ipv6_address(value)
+    return _validate_ipv4_address(value) or _validate_ipv6_address(value)
 
 
-class RegexValidator(object):
+def regex_check(address: EmailAddress) -> bool:
```

Comment:
What was the reason for removing RegexValidator? I don't see it maybe you could explain?

Reply:
The reason was simply that the class has degenerated to containing only a __call__() method and nothing else, so it can be replaced by a plain function. Is there any reason to wrap this function into a class I didn't see?